### DISCUSSION TITLE
Thank you! Lifesaver

### DISCUSSION THREAD

**Discussion by cmdcolin** [garronej/tss-react#90] [General]
Just wanted to say thanks for this library! Lifesaver.

Context: our mui v4 codebase we had was getting pretty clunky, I tried doing an upgrade to mui v5 but the legacy makeStyles in @mui/styles was not working quite right in certain scenarios... I saw your comments on an article (https://dev.to/atonchev/material-ui-5-the-easiest-way-to-migrate-from-makestyles-to-emotion-1i9l) and after 1 day in, I pretty much wrapped up the whole mui v5 changes. definitely curious about https://github.com/mui/material-ui/pull/32067 

thanks so much for your work!

**Comment by garronej** [comment]
Thank you very much for taking the time to reach out.  
It really means a lot.  
> definitely curious about https://github.com/mui/material-ui/pull/32067  
I will tackle it eventually, it re motivate me to make time for it!  

If you want you can do something for me that I'll highly appreciate.  
Suggest my name to [this GitHub program](https://stars.github.com/program/).  But don't feel obligated to!
